setup.md

```diff
@@ -21,13 +21,25 @@ to get a JupyterLab session with everything included. Once JupyterLab opens, cli
 If you want to install some of these packages on your own or your lab's computer, I recommend [Miniforge](https://github.com/conda-forge/miniforge) (or Anaconda/Miniconda with the [conda-forge channel](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge)). This method also provides a way to [install ROOT in the same environment](https://github.com/conda-forge/root-feedstock#readme). After setting that up, Uproot (for instance) could be installed as
 
 ~~~
-conda install uproot awkward   # ... others?
+conda install numpy uproot4 awkward hist particle hepunits matplotlib boost-histogram iminuit zfit vector fastjet
 ~~~
 {: .language-bash}
 
 If one of the above-mentioned packages is not available on conda-forge (yet), it can be installed in the same environment using pip:
```

Comment:
The conda file also takes care of these packages (see the `pip` bit at the bottom). So I would delete everything about here and just write "Alternatively, you can install all required packages locally with `pip`: Take a look at `environment.yml` for a list (this includes all dependencies listed under the `pip` key and some of the dependencies listed above)."